cmd/istioctl/inject.go

```diff
@@ -107,10 +107,20 @@ Example usage:
 )
 
 func init() {
+	defaultHub := os.Getenv(inject.DefaultHubEnvVar)
```

Comment:
Any concern about this being an implicit change based on an environment variable that could be different/non-existent based on which terminal sessions a user runs `istioctl`? For example, I frequently have multiple windows open (w/tmux) and run interactive commands in one window and monitor output in another. Sometimes I accidently switch the window I'm using for interactive commands. In this case the envvar might not be set but inject would proceed without printing an error and it might take me some time to figure out I'm using the wrong injected version. I'm wondering if it might be better to remove the default values altogether and explicitly require the user to specify hub/tag with flags or envvars. Thoughts?